webtau-feature-testing/examples/scenarios/graphql/Report.groovy

```diff
@@ -0,0 +1,78 @@
+package scenarios.graphql
+
+import org.testingisdocumenting.webtau.report.ReportDataProviders
+import org.testingisdocumenting.webtau.reporter.WebTauReport
+
+class Report {
+    static void generateReport(WebTauReport report) {
+        def additionalData = [:]
+        ReportDataProviders.provide(report.tests)
+            .map { it.toMap() }
+            .forEach { additionalData.putAll(it) }
+
+        validateSkippedOps(additionalData.graphQLSkippedOperations as Set)
+        validateCoveredOps(additionalData.graphQLCoveredOperations as Set)
+        validateOperationStatistics(additionalData.graphQLOperationTimeStatistics)
+        validateCoverageSummary(additionalData.graphQLCoverageSummary)
+    }
+
+    static void validateSkippedOps(Set skippedOps) {
+        skippedOps.should == [
+            [
+                name: 'uncomplete',
```

Comment:
uncomplete? incomplete?

Reply:
It's undoing the completion 🤷   I don't think `incomplete` is a verb and given this is a mutation I figured it out to be a verb rather than noun (yes, I know it's debatable whether `uncomplete` is a real word though).

Reply:
nvm